City-of-Helsinki/drupal-helfi-kymp
# Updated content_translation_changed field translation.

How to test: 
- Checkout this branch and do `make drush-cim drush-cr`
- Login and go to https://helfi-kymp.docker.so/fi/admin/content/integrations/tpr-unit
- Check that the view works
